proficio_sandbox/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 2.6)
+project(libbarrett_sandbox)
+
+
+# By default, build programs with compiler optimizations and debugging symbols
+if (NOT CMAKE_BUILD_TYPE)
+   set(CMAKE_BUILD_TYPE "RelWithDebInfo")
+endif()
+
+
+## libbarrett
+find_package(Barrett REQUIRED)
+include_directories(${BARRETT_INCLUDE_DIRS})
+link_directories(${BARRETT_LIBRARY_DIRS})
+add_definitions(${BARRETT_DEFINITIONS})
+
+## curses
+find_package(Curses REQUIRED)
+include_directories(${CURSES_INCLUDE_DIR})
+
+
+function(add_programs)
+	foreach (prog ${ARGV})
+		add_executable(${prog} ${prog}.cpp)
+		target_link_libraries(${prog} ${BARRETT_LIBRARIES})
+	endforeach()
+endfunction()
+
+add_programs(
```

Comment:
Since we have not changed any of the sandbox apps, I believe this might come under regression testing. The regression testing will be performed after the code review. I assume lot of examples will fail because of the left/right transformation in the world-to-base transformation. We have to decide whether it is important to modify the sandbox apps to support the Proficio or if it is up to the users, given the limited user base that we have for Proficio.